### DISCUSSION TITLE
S3 storage set-up failing

### DISCUSSION THREAD

**Discussion by KCAmplify** [airbytehq/airbyte#45346] [Deployment] [answered]
Hello,

I followed the set-up instruction here: https://docs.airbyte.com/deploying-airbyte/integrations/storage
What I wanted to do was for Airbyte to not store anything new on the server (I noticed the storage usage kept going up), and utilize external database to manage states (I've successfully set that up) and use S3 for anything that it needs to store (such as logs).
Above instructions results in the following error message. How can I set it up to use external S3 bucket for anything other than state, which can be stored in external database?

Thank you.

`unable to install airbyte chart: unable to install helm: cannot patch "airbyte-abctl-server" with kind Deployment: Deployment.apps "airbyte-abctl-server" is invalid: [spec.template.spec.containers[0].env[44].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[45].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')] && cannot patch "airbyte-abctl-worker" with kind Deployment: Deployment.apps "airbyte-abctl-worker" is invalid: [spec.template.spec.containers[0].env[46].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[47].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')] && cannot patch "airbyte-abctl-workload-launcher" with kind Deployment: Deployment.apps "airbyte-abctl-workload-launcher" is invalid: [spec.template.spec.containers[0].env[57].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[58].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]`

**Comment by amrael** [comment, marked as answer]
Downgrading the helm chart version to 0.524.0 by adding the option `--chart-version 0.524.0` to abctl local install command worked.

**Reply by KCAmplify** [reply]
Thank you. This seems to be working at the moment. I see files in S3 bucket. Monitoring to see if I don't have an increase on the local instance storage utilization, and will need to run few more tests, then will come back and mark this.

**Reply by KCAmplify** [reply]
I'm marking this as an answer as it does seem like there are files being written in S3 bucket specified and I can start the system. However, it does continue to increase the local container `airbyte-abctl-control-plane` size very slowly, so not everything is being propagated to s3 bucket I think.
